README.md

```diff
@@ -6,35 +6,42 @@
 # Tic-Tac-Toe on Solana
 
 This project demonstrates how to use the [Solana Javascript API](https://github.com/solana-labs/solana-web3.js)
-to implement an interactive tic-tac-toe game between two users.
+to build, deploy, and interact with programs on the Solana blockchain, implementing an interactive tic-tac-toe game between two users.
+To see the final product, go to https://solana-example-tictactoe.herokuapp.com/ and wait for another player to join.
+(Simply direct a 2nd browser window to the web app to play against yourself.)
 
-* The on-chain portion is a BPF program written in C, see `program-bpf/`
-* Command-line and web front-ends are provided under `src/`
+The project comprises:
```

Comment:
```suggestion
The project comprises of:
```

Reply:
Grammar says either "comprises" or "is comprised of"

Reply:
Oh probably, I almost failed english :)